src/Resources/app/administration/src/overrides/sw-data-grid-inline-edit/template.twig

```diff
@@ -1,21 +1,21 @@
 {% block sw_data_grid_inline_edit_type_unknown %}
```

Comment:
I think the changes to this file should be reverted, as only indentation changes.